src/index.html

```diff
@@ -7,7 +7,120 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="body body--child">
+      <div class="stars stars--0">
+        <div
+          class="stars__star"
+        ></div>
+        <div
+          class="stars__star"
+        ></div>
+        <div
+          class="stars__star"
+        ></div>
+        <div
+          class="stars__star"
+        ></div>
+        <div
+          class="stars__star"
+        ></div>
```

Comment:
```suggestion
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```
you have only one modifier, so why did you need to transfer code on new line?

Reply:
fix all code below